src/abstractdataframe/abstractdataframe.jl

```diff
@@ -365,20 +365,20 @@ support such objects (and not only vectors), and cannot access missing values.
 
 ```julia
 df = DataFrame(i = 1:10, x = rand(10), y = rand(["a", "b", "c"], 10))
-describe(df)
-describe(df, :all)
-describe(df, :min, :max)
-describe(df, :min, :sum => sum)
+describe([io,] df)
+describe([io,] df, :all)
+describe([io,] df, :min, :max)
+describe([io,] df, :min, :sum => sum)
 ```
 
 """
-StatsBase.describe(df::AbstractDataFrame, stats::Union{Symbol, Pair{Symbol}}...) =
+DataAPI.describe(io::IO, df::AbstractDataFrame, stats::Union{Symbol, Pair{Symbol}}...) =
```

Comment:
Do we have to quality here and below with `DataAPI.` prefix? I think it can be omitted. Right?

Reply:
We can omit, but I personally like to always explicitly qualify, just to be explicit about where this method is being defined.